src/libs/ReportUtils.js

```diff
@@ -229,7 +229,13 @@ function canEditReportAction(reportAction) {
  * @returns {Boolean}
  */
 function isSettled(reportID) {
-    return !lodashGet(allReports, [`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, 'hasOutstandingIOU']);
+    const report = lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {});
+    return report && !report.hasOutstandingIOU && !report.isWaitingOnBankAccount;
```

Comment:
Report will default to {} which is always truthy

```suggestion
    return !report.hasOutstandingIOU && !report.isWaitingOnBankAccount;
```